src/QueryResult.php

```diff
@@ -0,0 +1,31 @@
+<?php declare(strict_types=1);
+
```

Comment:
can we get some documentation for this purpose? what is the "result"?

Reply:
not sure yet ;)